### PR TITLE
update forms to v1.1.0

### PR DESCRIPTION
**Pull request by sarahvc**
http://uxframework.pearson.com/c/forms/

### CODE REVIEW

File: demo/demoPages/styles/FormsPage.js [hunk 1]
```diff
@@ -1,93 +1,293 @@
...
+                  <label className="pe-textLabelInput__label" htmlFor="d">Last Name</label>
+                  <input type="text" className="pe-textInput--basic" id="d" placeholder="Last Name"/>
+                  <span className="pe-input--info_message"></span>
+                  <span className="pe-input--error_message">This is a field level error message.</span>
+                </div>
+                <div className="pe-form--group pe-form--inline-group">
+                  <DatePicker
+                    id              = "someGiantId1"
+                    className       = "pe-form--inline-element"
+                    dateFormat      = "dd/mm/yyyy"
```
Comment (mhomolak) [inline demo/demoPages/styles/FormsPage.js]: I know this is just a demo page but could we have the same format of dates in the date picker if we're using them as an example?  We're never going to have different formats for a start and end date.

**Comment by StommePoes** [comment]
Wait, Mike, how can she do that? She did a PR, looks like others did a bunch of changes...

**Comment by mhomolak** [comment]
She got all of the changes that had been done since the branch was started while doing merges.  I thought a fresh branch could solve that.  If not disregard.

File: demo/demoPages/styles/FormsPage.js (hunk 1 above)
Comment (sarahvc) [reply demo/demoPages/styles/FormsPage.js]: @mhomolak Good catch. I just push another commit to fix this.

**Comment by mhomolak** [comment]
Don't worry about it.  Just proceed as you were.

**Comment by umahaea** [comment]
All approved. good to merge.

**Pull request merged**